code/controllers/subsystems/supply.dm

```diff
@@ -318,6 +318,8 @@ var/datum/controller/subsystem/cargo/SScargo
 /*
 	Getting items, categories, suppliers and shipments
 */
+/datum/controller/subsystem/cargo/proc/get_order_number()
```

Comment:
This sounds like it would get the order number of a given order.

Instead it returns the next order's number. Or something to that effect. Basically, bad name.